components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/model/ParRequestDO.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.par.model;
+
+import java.util.Map;
+
+/**
+ * Contains PAR attributes that will be retrieved from the database.
+ */
+public class ParRequestDO {
+
+    private Map<String, String> params;
+    private long scheduledExpiryTime;
```

Comment:
SchduledExpiry doesn't sit well with me. How about expiresIn/expiryTime to suit the requirement.

Reply:
Lets address in next iteration.

Reply:
Addressed